ITunesSearch/ITunesSearch/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  ITunesSearch
+//
+//  Created by Taylor Lyles on 5/14/19.
+//  Copyright © 2019 Taylor Lyles. All rights reserved.
+//
+
```

Comment:
You don't need this file. You are not using it so you can delete it and move to trash.